test/express.ts

```diff
@@ -1,17 +1,18 @@
-const { app } = require('../bin/www');
-const chai = require('chai');
-const chaiHTTP = require('chai-http');
-const { expect } = chai;
+import server from "../bin/server";
```

Comment:
Let's put relative imports after package imports (probably, eslint [rule](https://eslint.org/docs/rules/sort-imports) could be useful)